google-cloud-tools-plugin/testSrc/com/google/gct/idea/debugger/ServerToIDEFileResolverTest.java

```diff
@@ -0,0 +1,80 @@
+package com.google.gct.idea.debugger;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.intellij.openapi.project.Project;
+import com.intellij.psi.PsiClass;
+import com.intellij.psi.PsiFile;
+import com.intellij.psi.PsiJavaFile;
+import com.intellij.testFramework.fixtures.JavaCodeInsightFixtureTestCase;
+
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link com.google.gct.idea.debugger.ServerToIDEFileResolver}.
+ */
+public class ServerToIDEFileResolverTest extends JavaCodeInsightFixtureTestCase {
+  private Project project;
+  private PsiClass class1;
+  private PsiClass class2;
+  private PsiFile file1;
+  private PsiFile file2;
+
+  @Override
+  public void setUp() throws Exception {
+    super.setUp();
+    project = this.getProject();
+
+    class1 = this.myFixture.addClass("package com.java.pkg; class Class {}");
+    class2 = this.myFixture.addClass("package com.java.pkg; class ClassTest {}");
+  }
+
+  @Override
+  public void tearDown() throws Exception {
+    super.tearDown();
+  }
+
+  @Test
+  public void testGetCloudPathFromJavaFile() {
+    PsiJavaFile psiJavaFile = mock(PsiJavaFile.class);
+    when(psiJavaFile.getPackageName()).thenReturn("com.java.package");
+    when(psiJavaFile.getName()).thenReturn("Class.java");
+    assertEquals(ServerToIDEFileResolver.getCloudPathFromJavaFile(psiJavaFile),
+        "com/java/package/Class.java");
+  }
+
+  // When searching for full file system path.
+  @Test
+  public void testGetFileFromPath_fullPath() {
```

Comment:
I'd rather we @Ignore the test rather than comment out.  You should file an issue to fix.